hack/generate-changelog.sh

```diff
@@ -64,11 +57,16 @@ while [[ $# -gt 0 ]]; do
 done
 
 # Fetch the latest tags from the remote
-git fetch $REMOTE --tags
+git fetch --tags
```

Comment:
this means, this script is never to be run manually

Reply:
Can you confirm what `git fetch --tags` does on your environment.

Reply:
I see, I am wrong.